docs/installation_guide.md

```diff
@@ -100,19 +100,24 @@ The parameters are described below - note that at the very least, you will need
 
 - **harbor_admin_password**: The administrator's initial password. This password only takes effect for the first time Harbor launches. After that, this setting is ignored and the administrator's password should be set in the Portal. _Note that the default username/password are **admin/Harbor12345** ._
 
-
-
 - **database**: the configs related to local database
-  - **password**: The root password for the PostgreSQL database used for **db_auth**. _Change this password for any production use!_
+  - **password**: The root password for the PostgreSQL database. Change this password for any production use.
+  - **max_idle_conns**: The maximum number of connections in the idle connection pool. If <=0 no idle connections are retained. The default value is 2.
+  - **max_open_conns**: The maximum number of open connections to the database. If <= 0 there is no limit on the number of open connections. The default value is 0 (unlimited), except for Postgres, for which the default number of connections is 100.
```

Comment:
**max_open_conns**: The maximum number of open connections to the database. If <= 0 there is no limit on the number of open connections. Note that the default value is 100 for the max connections of harbor db is 100 and if it is not configured the value is 0.

Reply:
Fixed in https://github.com/goharbor/harbor/pull/9005/commits/55de600b2ae0a7bbfb64d9640ccf9c2413dabb47.